pkg/tracker/porttracker.go

```diff
@@ -16,97 +16,70 @@ limitations under the License.
 package tracker
 
 import (
-	"sync"
-
-	"github.com/Masterminds/log-go"
 	"github.com/docker/go-connections/nat"
 	"github.com/rancher-sandbox/rancher-desktop-agent/pkg/forwarder"
 	"github.com/rancher-sandbox/rancher-desktop-agent/pkg/types"
 )
 
+type Tracker interface {
```

Comment:
Please move this interface into its own file (since it's implemented by both `APITracker` and `PortTracker`), and add a doc comment block + doc comments on each of the methods.

We should be able to move `NetTracker` to the same file.